feedback-module/src/components/Module.js

```diff
@@ -101,6 +104,30 @@ function Module({ pageTitle }) {
     return checkedFields;
   };
 
+  //Checks if at least one checkbox was checked - returns true if yes false if no
+  const checkboxValidated = () => {
+    return checkedFields.some((field) => field.checked);
+  };
+
+  //Checks if all the required fields have been completed - returns true if yes false if no
+  const inputsValidated = () => {
+    let validated = true;
+    let errorInputs = [];
+    inputQuestions.forEach((question, index) => {
+      en(screen.textInputs).map((input) => {
```

Comment:
Since it seems like the purpose of looping through both `inputQuestions` and `en(screen.textInputs)` is just to figure out if a field is required, could we just initialize the objects in inputQuestion to have a required field?

So when we're initializing `inputQuestions` in the `useEffect` it could look something like this:
```
return { question: question.text, answer: "", required: question.required };
```
Then, instead of having both the forEach and map here we can just do
```
inputQuestions.forEach((question) => {
      if (
        question.required &&
        question.answer === ""
      ) {
        errorInputs.push(question.question), (validated = false);
      }
    });
```

Reply:
Much cleaner I like it. The only thing is does that mean we are going to be sending the required field in the JSON to the backend then?